vcsutils/consts.go

```diff
@@ -27,7 +27,9 @@ func (v VcsProvider) String() string {
 type WebhookEvent string
 
 const (
-	PrCreated WebhookEvent = "PrCreated"
-	PrEdited  WebhookEvent = "PrEdited"
-	Push      WebhookEvent = "Push"
+	PrClosed WebhookEvent = "PrClosed"
```

Comment:
I am afraid `PrClosed` might be misunderstood (as it could include merged and rejected PRs). `PrRejected` may be a better name.